WordPress/changing_mysql_database_password.md

```diff
@@ -13,14 +13,12 @@ Please note that the Application Settings related to database connection are **p
 |    DATABASE_USERNAME           |
 |    DATABASE_PASSWORD           |
 
+## Changing MySQL Database Password
 
+First, go to the MySQL resource corresponding to your WordPress deployment, and click on **Reset Password** option as shown below. Now enter the new password and click on Save. Wait until the action is completed.
 
-### Changing MySQL Database Password
-
-First, go to the MySQL resource corresponding to your WordPress deployment, and click on **Reset Password** option as shown below. Now enter the new password and click on Save. Wait until the action is completed. 
+Then navigate to the Configuration section of your AppService and update the **DATABASE_PASSWORD** Application Settings in your AppService. Once you update the value, click on Save and wait for app to get restarted.
 
-Then navigate to the Configuration section of your AppService and update the **DATABASE_PASSWORD** Application Settings in your AppService. Once you update the value, click on Save and wait for app to get restarted. 
+![Password Rest](./media/changing_mysql_password_1.png)
 
-<br>
```

Comment:
@rabollin we should not be in-lining html, this was a common mistake throughout the repo

Reply:
Agreed